bin/juxtaposer/tester/db/mssql/mssql.go

```diff
@@ -0,0 +1,107 @@
+package mssql
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"net/url"
+
+	_ "github.com/denisenkom/go-mssqldb"
+
+	"github.com/cyberark/secretless-broker/bin/juxtaposer/tester/api"
+	sql_db_tester "github.com/cyberark/secretless-broker/bin/juxtaposer/tester/db/sql"
+)
+
+type MssqlTester struct {
+	sql_db_tester.SqlDatabaseTester
+}
+
+func NewTester() (api.DbTester, error) {
+	tester := &MssqlTester{}
+
+	return tester, nil
+}
+
+// sqlserver://username:password@host:port?database=master&param2=value
+
+func (tester *MssqlTester) GetQueryMarkers(length int) string {
+	markerString := ""
+	for markerIndex := 1; markerIndex <= length; markerIndex++ {
+		markerString = markerString + fmt.Sprintf("@p%d, ", markerIndex)
+	}
+
+	return markerString[:len(markerString)-2]
```

Comment:
Instead of assembling the markerString and then truncating the trailing comma, you could use strings.Join() with a ", " separator:
```
func (tester *MssqlTester) GetQueryMarkers(length int) string {
	markers := []string{}
	for markerIndex := 1; markerIndex <= length; markerIndex++ {
		markers = append(markers, fmt.Sprintf("@p%d", markerIndex))
	}
	return strings.Join(markers, ", ")
}
```